handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -256,6 +256,13 @@ public static boolean isAlpnSupported() {
     }
 
     /**
+     * Returns {@code true} if the used version of OpenSSL supports OCSP stapling.
+     */
+    public static boolean isOcspSupported() {
+      return version() >= 0x10002000L;
```

Comment:
OK, LibreSSL works. 

BoringSSL doesn't work (fails gracefully with my OCSP is not supported exception) but that is because... It's either getting built with `OPENSSL_NO_OCSP` or `OPENSSL_VERSION_NUMBER` is something very different than expected. Checking... :thinking: